packages/angular/cli/src/commands/mcp/tools/build_spec.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { execSync, spawnSync } from 'child_process';
+import { BUILD_TOOL, BuildToolInput, runBuild } from './build';
+import { McpToolContext } from './tool-registry';
+
+// Mock the execSync function
+const mockedExecSync = jasmine.createSpy('execSync');
+
+// Replace the actual execSync with our mock
+Object.defineProperty(require('child_process'), 'execSync', {
+  value: mockedExecSync,
+});
+
+describe('Build Tool', () => {
+  beforeEach(() => {
+    mockedExecSync.calls.reset();
+  });
+
+  it('should handle a successful build and extract the output path', () => {
+    const buildLogs =
+      'Build successful!\nSome other log lines...\nOutput location: dist/my-cool-app';
```

Comment:
Consider: I'm a little worried that changes to `ng build` logging could unintentionally break this. Ideally we would have at least one test which _actually_ runs `ng build` to verify.

However there's maybe some simpler solution of just finding wherever `Output location:` is in source code and adding a comment that this is a load-bearing detail for the MCP server. Hopefully if we ever changed that we would at least notice the comment.

Alternatively, I wonder if it's worth tweaking the build output to make this easier. Could we (maybe gated on a private environment variable) adjust `ng build` output with something more explicit like:

```
BUILD COMPLETE: {
  "outputLocation": "dist/my-cool-app"
}
```

And then parse it in a slightly more structured fashion?